styles/about_me.css

```diff
@@ -0,0 +1,93 @@
+.about__me > div {
+  position: relative;
+  padding-bottom: 50px;
+  padding-top: 50px;
+}
+
+.about__me > div::before {
+  content: "";
+  left: -20%;
+  z-index: -1;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  background-color: #fafaff;
+}
+
+.gray__bg > h2 {
+  display: flex;
+  justify-content: flex-start;
+  font-family: "Ubuntu", sans-serif;
+  padding-left: 182px;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 37px;
+  letter-spacing: 0.02em;
+  color: #222222;
+}
+
+.main__box {
+  background-color: white;
+  box-shadow: 6px 6px 20px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.main__box > div > img {
+  border-radius: 50%;
+  margin-left: 50px;
+  margin-bottom: 85px;
+}
+
+.person__data {
+  width: 40%;
+  margin-left: 20px;
+  padding-left: 30px;
+}
+
+.person__data > p {
+  font-family: "Raleway", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  font-style: normal;
+  line-height: 29px;
+  letter-spacing: 0.02em;
+  color: #222222;
+}
+
+.person__data > p > span {
+  color: #ffc53d;
+  font-family: "Ubuntu", sans-serif;
+  font-weight: 700;
```

Comment:
Tutaj nie musimy teoretycznie tego już przekazywać, dostanie to od rodzica już

Reply:
Niestety nie, po usunięciu tego kodu znikają style które muszą tam być (żółty tekst w danych personalnych, zgodnie z figmą też ma mieć on inny font niż .person__data). Usunęłam font weight - to rzeczywiście ma od rodzica